File: shuffle sort/shuffleSort.py
def shuffleSort(arr):
    start = 0
    end = len(arr) - 1
    while start < end:
        for i in range(start, end):
            if arr[i] > arr[i + 1]:
                arr[i], arr[i + 1] = arr[i + 1], arr[i]
        end-= 1
        for i in range(end, start, -1):
            if arr[i] > arr[i + 1]:
                arr[i], arr[i + 1] = arr[i + 1], arr[i]

        start += 1

File: shuffle sort/shuffleSort.js
// P.S. start and finish aren't important part. These variables contain information when algorithm started and finished
        function shuffleSort(arr) {
            let start = 0,
                end = arr.length - 1,
                startTime = Date.now();

            while (start < end) {
                for (let i = start; i <= end; ++i) {
                    if (arr[i] > arr[i + 1]) {
                        [arr[i], arr[i + 1]] = [arr[i + 1], arr[i]];
                    }
                }
                end--;
                for(let i = end; i >= start; --i){
                    if(arr[i] > arr[i + 1]){
                        [arr[i], arr[i + 1]] = [arr[i + 1], arr[i]];
                    }
                }
                start ++;
            }
            let endTime = Date.now();
            timer = endTime - startTime + 'ms';
        }
